### DISCUSSION TITLE
Logging (using `Uno.Extensions.Logging` in WinUI)

### DISCUSSION THREAD

**Discussion by BertrandDeSaintBezier** [unoplatform/uno#13676] [Q&A] [answered]
Hi !

When running a project in Debug Mode, logs are output to the console. I was wondering if these logs were also written to disk in the context of a WinUI/Windows build when running a deployed instance of application (aka, not from VS), or if this functionality needs to be implemented manually ?

Thanks !

**Comment by jeromelaban** [comment, marked as answer]
We have documentation for logging here https://platform.uno/docs/articles/logging.html#logging-in-uno, which is based on the [Microsoft.Extensions.Logging](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line)
